packages/filesystem/src/browser/filesystem-frontend-contribution.ts

```diff
@@ -301,13 +304,15 @@ export class FileSystemFrontendContribution implements FrontendApplicationContri
             }
             if (!deleted) {
                 widget.title.label += this.deletedSuffix;
+                this.editorFileWasChangedEmitter.fire({ editor: widget, type: FileChangeType.DELETED });
```

Comment:
Shouldn't be a problem to do so, though there might be a fair bit of code duplication since the `file-system-frontend-contribution` has a quite a bit of filtering/checks to match up the fileChangeEvents with the open NavigatableWidgets. I thought that adding an event would result in the fewer lines :)

Let me know what you think, happy to make the change either way